src/cfnlint/data/CloudSpecs/ap-northeast-1.json

```diff
@@ -56529,12 +56603,6 @@
           "Required": false,
           "UpdateType": "Immutable"
         },
-        "Name": {
-          "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-imagebuilder-component.html#cfn-imagebuilder-component-name",
-          "PrimitiveType": "String",
-          "Required": true,
-          "UpdateType": "Immutable"
-        },
```

Comment:
lost the `Name` property from [5 ImageBuilder resource types](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/AWS_ImageBuilder.html) in this week's [resource specification](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html)

same reason as [`AWS::StepFunctions::Activity.Name`](https://github.com/aws-cloudformation/cfn-python-lint/pull/1715#discussion_r496397195)

[(incorrectly marking non-`readOnlyProperties` as `readOnlyProperties` for `GetAtt` support)](https://github.com/aws-cloudformation/aws-cloudformation-coverage-roadmap/issues/68#issuecomment-665243220)

Reply:
@PatMyron good eye.  Should we patch this in?

Reply:
Patched.